### PR TITLE
removing the mention of '/api/' from API paths in examples CE-587

### PR DESCRIPTION
**Pull request by vlad-bluescape**

### CODE REVIEW

File: site-templates/pages/what-is-new-in-v3.gomd [hunk 1]
```diff
@@ -29,11 +29,11 @@ The main differences can be grouped in 3 sections:
     <br /><br >
     Please note the following V3 REST APIs that still use the <b>PUT</b> verb to make a change:
     <ul>
-      <li>Create Asset Upload: <pre>/api/v3/workspaces/{workspace}/assets/uploads/{id}/error</pre></li>
-      <li>Progress Asset Upload to processing: <pre>/api/v3/workspaces/{workspace}/assets/uploads/{id}</pre></li>
-      <li>Update the Member's Role:<pre>PUT /api/v3/organizations/{organizationId}/members/{memberId}/role</pre></li>
-      <li>Update Avatar: <pre>PUT /api/v3/users/me/avatar</pre></li>
-      <li>Favourite a workspace: <pre>PUT /api/v3/users/me/favorites/workspaces/{workspaceId}</pre></li>
+      <li>Create Asset Upload: <pre>PUT /v3/workspaces/{workspace}/assets/uploads/{id}/error</pre></li>
```
Comment (Jackychen-bluescape) [inline site-templates/pages/what-is-new-in-v3.gomd:32]: Why does "Create" correspond to "PUT"?

Comment (vlad-bluescape) [reply site-templates/pages/what-is-new-in-v3.gomd:32]: Just following the documentation:
image

Comment (vlad-bluescape) [reply site-templates/pages/what-is-new-in-v3.gomd:32]: yes, does not make sense, but that is how the documentation states it.

Comment (Jackychen-bluescape) [reply site-templates/pages/what-is-new-in-v3.gomd:32]: Ah, it's part of the error endpoint. I suppose it's updating the error message for failed uploads.

**Review by Jackychen-bluescape — APPROVED** [review]
LGTM.

**Pull request merged**